app/Http/routes.php

```diff
@@ -168,20 +154,90 @@
         return view('dashboard.pages.edit_episode');
     });
 
-    Route::get('/channels', function () {
-        return view('dashboard.pages.view_channels');
-    });
-
-    Route::get('/channel/create', function () {
-        return view('dashboard.pages.create_channel');
-    });
+/*
+/-------------------------------------------------------------------------------
+/ Admin User
+/-------------------------------------------------------------------------------
+*/
+    Route::get('/users', [
+        'uses' => 'UserController@index',
+        'as'   => '/users'
+    ]);
+
+    Route::get('/user/edit/{id}', [
+        'uses' => 'UserController@editView',
+        'as'   => '/user/edit/{id}'
+    ]);
+
+    Route::post('/user/edit', [
+        'uses' => 'UserController@update',
+        'as'   => '/user/edit'
+    ]);
+
+    Route::get('/user/create', [
+        'uses' => 'UserController@show',
+        'as'   => '/user/create'
+    ]);
+
+    Route::post('/user/create', [
+        'uses' => 'UserController@createInvite',
+        'as'   => '/user/create'
+    ]);
 
-    Route::get('/channel/edit', function () {
-        return view('dashboard.pages.edit_channel');
-    });
+/*
+/-------------------------------------------------------------------------------
+/ Admin Channel
+/-------------------------------------------------------------------------------
+*/
+    Route::get('/channels', [
+        'uses' => 'ChannelController@index',
+        'as'   => '/channels'
+    ]);
+
+    Route::get('/channel/edit/{id}', [
```

Comment:
It should be /channel/{id}/edit, that's more RESTFUL.


Please make adjustments to all faulting routes in this file.